scripts/source_code.py

```diff
@@ -13,7 +13,10 @@
 def clang_get_func_code(
     file_path: str,
     function_name: str,
-    get_node_name: Callable[[clang.cindex.Cursor], str] = lambda node: node.spelling,
+    get_node_name: Callable[
+        [clang.cindex.Cursor], str
+    ] = lambda node, name: node.spelling
+    == name,
```

Comment:
The type of this parameter is wrong. Please check https://docs.python.org/3/library/typing.html#annotating-callable-objects